src/options.ts

```diff
@@ -86,6 +86,24 @@ const vueOptions: BeautifierLanguageOptions = {
 };
 delete vueOptions.printWidth;
 
+const yamlOptions: BeautifierLanguageOptions = {
+  bracketSpacing: "object_curly_spacing",
+  proseWrap: "wrap_prose",
+  singleQuote: [
```

Comment:
This block can be simplified to

```suggestion
singleQuote: scriptOptions.singleQuote,
```